runner/tasks.py

```diff
@@ -92,6 +92,12 @@ def process_triggers():
                     if operator_run.percent_runs_finished == 100.0:
                         operator_run.complete()
 
+            if operator_run.percent_runs_finished == 100.0:
```

Comment:
I think this should go in a conditional checking whether there are any triggers.

Reply:
Yes. That is why I added it. My second OperatorRun was hanging in RUNNING state.

Reply:
I meant a conditional to check if the operator run has any triggers associated with it, but actually i think we should keep this and remove all other `operator_run.complete/fail()`

Reply:
@aef- Something like this?